openassessment/xblock/data_conversion.py

```diff
@@ -218,3 +218,30 @@ def make_django_template_key(key):
         basestring
     """
     return key.replace('-', '_')
+
+
+def verify_assessment_parameters(func):
+    """
+    Verify that the wrapped function receives the given parameters.
+
+    Used for the staff_assess, self_assess, peer_assess functions and uses their data types.
+
+    Args:
+        func - the function to be modified
+
+    Returns:
+        the modified function
+    """
+    def verify_and_call(instance, data, suffix):
+        # Validate the request
+        if 'options_selected' not in data:
+            return {'success': False, 'msg': instance._('Must provide options selected in the assessment')}
```

Comment:
"You must provide the required options for this assessment." (?)
@cahrens Does this work? I'm not sure what "options selected in the assessments" refers to.